<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <arg name="dataset" default="newer_college/01_short_experiment" doc="Dataset and sequence name."/>
    <arg name="dataset_path" default="" doc="Override for dataset path, optional."/>
    <arg name="dataset_poses_path" default="" doc="Override for dataset poses path, optional."/>

    <node pkg="depth_correction" type="reconstruction_eval" name="reconstruction_eval" output="screen">
        <rosparam subst_value="true">
            dataset: '$(arg dataset)'
            dataset_path: '$(arg dataset_path)'
            dataset_poses_path: '$(arg dataset_poses_path)'
            map_topic: 'cloud'
            rate: 0.2
        </rosparam>
    </node>
</launch>
